src/components/Header/Header.jsx

```diff
@@ -1,12 +1,12 @@
 import * as React from "react"
 import "./Header.css"
 
-export function Header(props) {
+export function Header({info}) {
```

Comment:
Nit: IMO, I think separating out props into title, tagline, and description would be easier to understand since this component is for displaying information.